tests/examplefiles/groovy/test.gradle.output

```diff
@@ -88,7 +88,7 @@
 ' '           Text
 'String'      Name
 ' '           Text
-'getWorldString' Name.Function
+'getWorldString' Name
```

Comment:
This should be detected as `Name.Function` and not `Name`, but I think this is not a big problem considering it's much more annoying for keywords to be detected as functions (see `test.groovy.output` fixes, below).